app/routes/goal_routes.py

```diff
@@ -0,0 +1,96 @@
+from app import db
+from app.models.goal import Goal
+from app.models.task import Task
+from flask import Blueprint, request, jsonify, make_response
+from app.routes.task_routes import validate_id
+
+goal_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+# ================================
+# Create one goal 
+# ================================
+
+@goal_bp.route("", methods=["POST"])
+def create_goal():
+    request_body = request.get_json()
+
+    try:
+        new_goal = Goal(title=request_body["title"])
+    except KeyError:
+        return make_response({"details": "Invalid data"}, 400)
+
+    db.session.add(new_goal)
+    db.session.commit()
+
+    return jsonify({"goal": new_goal.to_json()}), 201
```

Comment:
Since you are returning a dictionary, you shouldn't need to use `jsonify` here, so you could replace this return statement with `return {"goal": new_goal.to_json()}, 201`.

I wager there's other places where you used `jsonify` like this, but I won't point them all out to avoid repeating myself.